rpc/legacyrpc/methods.go

```diff
@@ -1713,6 +1719,9 @@ func signRawTransaction(icmd interface{}, w *wallet.Wallet, chainClient *chain.R
 		if err != nil {
 			return nil, err
 		}
+		if result == nil {
+			return nil, errUtxoSpent
+		}
```

Comment:
ok cool - i guess ideally we could change Receive to return an error if the result is nil and then dont need to explicitly handle this. but i see we already do this special case handling in lnd for all calls to `Receive` done via `GetTxOut`.